client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/thirdpartyclasses/MicronautConfigurationReplacement.java

```diff
@@ -0,0 +1,29 @@
+package org.evomaster.client.java.instrumentation.coverage.methodreplacement.thirdpartyclasses;
+
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.Replacement;
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.ThirdPartyMethodReplacementClass;
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.UsageFilter;
+import org.evomaster.client.java.instrumentation.shared.ReplacementType;
+import org.evomaster.client.java.instrumentation.staticstate.ExecutionTracer;
+import org.evomaster.client.java.utils.SimpleLogger;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class MicronautConfigurationReplacement extends ThirdPartyMethodReplacementClass {
+
+    private static final MicronautConfigurationReplacement singleton = new MicronautConfigurationReplacement();
+
+    @Override
+    protected String getNameOfThirdPartyTargetClass() {
+        return "io.micronaut.http.server.netty.configuration.NettyHttpServerConfiguration";
+    }
+
+    @Replacement(replacingStatic = false,
+            type = ReplacementType.TRACKER,
+            id = "micronautNettyServerConfiguration_class",
+            usageFilter = UsageFilter.ONLY_SUT)
```

Comment:
is this correct? shouldn't it be `UsageFilter.ANY`?  i mean, what if this method is called by another method in Micronaut? (with `ONLY_SUT` it would not work)

Reply:
Since the method replacement didn't work properly, made this modification for further testing. Earlier version has ONLY_SUT. Will revert back.